{"name": "Переход в ленту заказов со страницы логина", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: Список заказов не отобразился\nStacktrace:\n\tGetHandleVerifier [0x00ECFC03+61635]\n\tGetHandleVerifier [0x00ECFC44+61700]\n\t(No symbol) [0x00CF05D3]\n\t(No symbol) [0x00D3899E]\n\t(No symbol) [0x00D38D3B]\n\t(No symbol) [0x00D80E12]\n\t(No symbol) [0x00D5D2E4]\n\t(No symbol) [0x00D7E61B]\n\t(No symbol) [0x00D5D096]\n\t(No symbol) [0x00D2C840]\n\t(No symbol) [0x00D2D6A4]\n\tGetHandleVerifier [0x01154523+2701795]\n\tGetHandleVerifier [0x0114FCA6+2683238]\n\tGetHandleVerifier [0x0116A9EE+2793134]\n\tGetHandleVerifier [0x00EE68C5+155013]\n\tGetHandleVerifier [0x00EECFAD+181357]\n\tGetHandleVerifier [0x00ED7458+92440]\n\tGetHandleVerifier [0x00ED7600+92864]\n\tGetHandleVerifier [0x00EC1FF0+5296]\n\tBaseThreadInitThunk [0x75F15D49+25]\n\tRtlInitializeExceptionChain [0x77ABD03B+107]\n\tRtlGetAppContainerNamedObjectPath [0x77ABCFC1+561]", "trace": "self = <tests.test_constructor.TestConstructorFromLogin object at 0x000001DE7E160C30>\ndriver = <undetected_chromedriver.Chrome (session=\"1080f9864dbf3f0bddd356e7a9342b6b\")>\n\n    @allure.title(\"Переход в ленту заказов со страницы логина\")\n    def test_go_to_order_feed_from_login(self, driver):\n        with allure.step(\"Открываем страницу логина\"):\n            self.driver.get(f\"{BASE_URL}/login\")\n            WebDriverWait(self.driver, 10).until(\n                EC.visibility_of_element_located(LoginPageLocators.LOGIN_BUTTON)\n            )\n        with allure.step(\"Кликаем кнопку 'Лента заказов'\"):\n            self.main_page.go_to_order_feed()\n        with allure.step(\"Проверяем загрузку ленты заказов\"):\n            assert self.main_page.wait_for_url_contains(\"/feed\"), \"Не открылась лента заказов\"\n            WebDriverWait(self.driver, 10).until(\n                EC.visibility_of_element_located(OrderFeedLocators.ORDER_FEED_SECTION),\n                \"Заголовок ленты заказов не отобразился\"\n            )\n>           WebDriverWait(self.driver, 10).until(\n                EC.visibility_of_element_located(OrderFeedLocators.ORDER_LIST),\n                \"Список заказов не отобразился\"\n            )\n\ntests\\test_constructor.py:50: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"1080f9864dbf3f0bddd356e7a9342b6b\")>\nmethod = <function visibility_of_element_located.<locals>._predicate at 0x000001DE7E344540>, message = 'Список заказов не отобразился'\n\n    def until(self, method: Callable[[WebDriver], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: Список заказов не отобразился\nE       Stacktrace:\nE       \tGetHandleVerifier [0x00ECFC03+61635]\nE       \tGetHandleVerifier [0x00ECFC44+61700]\nE       \t(No symbol) [0x00CF05D3]\nE       \t(No symbol) [0x00D3899E]\nE       \t(No symbol) [0x00D38D3B]\nE       \t(No symbol) [0x00D80E12]\nE       \t(No symbol) [0x00D5D2E4]\nE       \t(No symbol) [0x00D7E61B]\nE       \t(No symbol) [0x00D5D096]\nE       \t(No symbol) [0x00D2C840]\nE       \t(No symbol) [0x00D2D6A4]\nE       \tGetHandleVerifier [0x01154523+2701795]\nE       \tGetHandleVerifier [0x0114FCA6+2683238]\nE       \tGetHandleVerifier [0x0116A9EE+2793134]\nE       \tGetHandleVerifier [0x00EE68C5+155013]\nE       \tGetHandleVerifier [0x00EECFAD+181357]\nE       \tGetHandleVerifier [0x00ED7458+92440]\nE       \tGetHandleVerifier [0x00ED7600+92864]\nE       \tGetHandleVerifier [0x00EC1FF0+5296]\nE       \tBaseThreadInitThunk [0x75F15D49+25]\nE       \tRtlInitializeExceptionChain [0x77ABD03B+107]\nE       \tRtlGetAppContainerNamedObjectPath [0x77ABCFC1+561]\n\nvenv\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py:101: TimeoutException"}, "steps": [{"name": "Открываем страницу логина", "status": "passed", "start": 1748199209647, "stop": 1748199211155}, {"name": "Кликаем кнопку 'Лента заказов'", "status": "passed", "start": 1748199211155, "stop": 1748199233219}, {"name": "Проверяем загрузку ленты заказов", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: Список заказов не отобразился\nStacktrace:\n\tGetHandleVerifier [0x00ECFC03+61635]\n\tGetHandleVerifier [0x00ECFC44+61700]\n\t(No symbol) [0x00CF05D3]\n\t(No symbol) [0x00D3899E]\n\t(No symbol) [0x00D38D3B]\n\t(No symbol) [0x00D80E12]\n\t(No symbol) [0x00D5D2E4]\n\t(No symbol) [0x00D7E61B]\n\t(No symbol) [0x00D5D096]\n\t(No symbol) [0x00D2C840]\n\t(No symbol) [0x00D2D6A4]\n\tGetHandleVerifier [0x01154523+2701795]\n\tGetHandleVerifier [0x0114FCA6+2683238]\n\tGetHandleVerifier [0x0116A9EE+2793134]\n\tGetHandleVerifier [0x00EE68C5+155013]\n\tGetHandleVerifier [0x00EECFAD+181357]\n\tGetHandleVerifier [0x00ED7458+92440]\n\tGetHandleVerifier [0x00ED7600+92864]\n\tGetHandleVerifier [0x00EC1FF0+5296]\n\tBaseThreadInitThunk [0x75F15D49+25]\n\tRtlInitializeExceptionChain [0x77ABD03B+107]\n\tRtlGetAppContainerNamedObjectPath [0x77ABCFC1+561]\n\n", "trace": "  File \"C:\\Users\\windw\\PycharmProjects\\Diplom\\Diplom_3\\tests\\test_constructor.py\", line 50, in test_go_to_order_feed_from_login\n    WebDriverWait(self.driver, 10).until(\n    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^\n        EC.visibility_of_element_located(OrderFeedLocators.ORDER_LIST),\n        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n        \"Список заказов не отобразился\"\n        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n    )\n    ^\n  File \"C:\\Users\\windw\\PycharmProjects\\Diplom\\Diplom_3\\venv\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 101, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "start": 1748199233219, "stop": 1748199244559}], "parameters": [{"name": "driver", "value": "'chrome'"}], "start": 1748199209647, "stop": 1748199244562, "uuid": "68631390-8f80-46a3-b0a9-678961d8ff4c", "historyId": "c7fc6ae2fa4ccdefdafcbb8b17b71ddb", "testCaseId": "e586d6ea6551d59a7239cf3b5da65576", "fullName": "tests.test_constructor.TestConstructorFromLogin#test_go_to_order_feed_from_login", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_constructor"}, {"name": "subSuite", "value": "TestConstructorFromLogin"}, {"name": "host", "value": "Gemini_78"}, {"name": "thread", "value": "19464-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_constructor"}]}